### ISSUE TITLE
Tests fail for different filesystems

### ISSUE THREAD

**Issue by purplecabbage** [adobe/aio-app-scripts#48]
expectAppFiles has deep expectations of what the root file system looks like.
The tests should not actually touch the file system at all.

```
  ● with local actions and no frontend › should cleanup generated files on SIGINT

    expect(received).toEqual(expected) // deep equality

    - Expected
    + Received

      Set {
        "actions",
        "manifest.yml",
        "package.json",
    +   "repos",
        "web-src",
      }

      163 |   if (files.has('Users')) files.delete('Users')
      164 |   if (files.has('home')) files.delete('home')
    > 165 |   expect(files).toEqual(expectedFiles)
          |                 ^
      166 | }
```

**Comment by moritzraho** [comment]
@purplecabbage thanks good catch!
Just to be clear all tests use a memfs and do not touch the real fs at all.
The problem here is when downloading the openwhisk jar which uses __dirname to build the output path, the test is sloppy indeed as it assumes that __dirname always starts with /Users/ or /home/, which in your case is not as it downloads the openwhisk jar to a path starting with /repos/
I think this can be solved by mocking __dirname. Will take a look now

**Comment by moritzraho** [comment]
should be fixed by #49, so closing, let me know if we should reopen
